### PR TITLE
Move code of conduct in README.md

### PR DESCRIPTION
**Pull request by youngar**
Move code of conduct from "What's the goal?" to "How Do I Interact With the Community?"

### CODE REVIEW

**Comment by youngar** [comment]
Forgot to skip ci in the commit message :disappointed:

File: README.md [hunk 1]
```diff
@@ -133,13 +128,15 @@ There are some active contribution projects underway right now:
...
-The community typically uses this list for project announcements and administrative
-discussions amongst committers.  Questions are welcome here as well.
+We are operating under the [Eclipse Code of Conduct][coc] to promote fairness, openness, and inclusion.
 
-* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels
-that interest you, ask questions, and receive answers from subject matter experts.
+[coc]: https://eclipse.org/org/documents/Community_Code_of_Conduct.php
 
+* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels that interest you, ask questions, and receive answers from subject matter experts.
+  that interest you, ask questions, and receive answers from subject matter experts.
```
Comment (mstoodle) [inline README.md]: some copied text here, it looks like?

File: README.md [hunk 2]
```diff
@@ -133,13 +128,15 @@ There are some active contribution projects underway right now:
 How Do I Interact With the Community?
 =====================================
 
-* Join the Eclipse OMR developer community [mailing list](https://accounts.eclipse.org/mailing-list/omr-dev).
-The community typically uses this list for project announcements and administrative
-discussions amongst committers.  Questions are welcome here as well.
+We are operating under the [Eclipse Code of Conduct][coc] to promote fairness, openness, and inclusion.
```
Comment (mstoodle) [inline README.md]: "We operate under" is stronger . This one's a nit, but since I'm reviewing....:)

File: README.md [hunk 3]
```diff
@@ -133,13 +128,15 @@ There are some active contribution projects underway right now:
...
 
-* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels
-that interest you, ask questions, and receive answers from subject matter experts.
+[coc]: https://eclipse.org/org/documents/Community_Code_of_Conduct.php
 
+* Join the Eclipse OMR community [Slack workspace](https://join.slack.com/t/eclipse-omr/shared_invite/enQtMzg2ODIwODc4MTAyLWFiMzZkNmNhODc5OTM0MjgwZDdjNzg5YTg5NzM0ZmEzNTIyMGViMjk1YjYwNzczYjYwODc4YTM5MDk0NjIxMjg).  You can join channels that interest you, ask questions, and receive answers from subject matter experts.
+  that interest you, ask questions, and receive answers from subject matter experts.
+* Join the Eclipse OMR developer community [mailing list](https://accounts.eclipse.org/mailing-list/omr-dev).
+  The community typically uses this list for project announcements and administrative
```
Comment (mstoodle) [inline README.md]: maybe want to add a "primarily" for project announcements ...  I think we want to clearly set people's expectations that the best places to engage us are via Slack and GitHub issues.

File: README.md (hunk 1 above)
Comment (youngar) [reply README.md]: Yeah, looks like i messed up my merge. Thanks for pointing it out and its fixed

File: README.md (hunk 2 above)
Comment (youngar) [reply README.md]: Thanks for the suggestion, I agree it is stronger

**Pull request merged**